plugin/Installer/DatabaseChanges.php

```diff
@@ -161,6 +161,7 @@ public function install()
 
 		if ($dbversion == 20) {
 			$this->updateCreateAddressFieldOfIntersetAndOwnerForm();
+            dbDelta( $this->getCreateQueryListviews() );
```

Comment:
Was there a change in the method `getCreateQueryListviews()` or what is the reason to have it run here again?

Reply:
This is still open